release-notes/release-notes-template.md

```diff
@@ -0,0 +1,55 @@
+## Introduction
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud < release-version >.
+
+Please see [component versions](https://anbox-cloud.io/docs/component-versions) for a list of updated components.
+
+## New features & improvements
+
+### Feature 1
+
+< Explain feature 1 >
+
+### Feature 2
+
+< Explain feature 2 >
+
+
+< Add feature sections as necessary >
+
+
+### Dashboard improvements
+
+< List enhancements to the dashboard here. You can screenshots to illustrate something new or enhanced. >
+
+
+### Other
+
+< In addition to the below placeholder points, list minor features that are part of the release but were not elaborated as separate sections >
+
+* Android security updates for < insert-month-year > (see Android Security Bulletin - < insert-month-year along with the link to bulletin > for more information).
+* The Android WebView has been updated to < insert-version-along-with-link >.
```

Comment:
This reminds me of one thing: looks like we forgot to mention the update to Android Webview in our 1.18.0 release note.

Reply:
I will update the release notes for 1.18.